doc/changelog/08-cli-tools/13876-coqc+no_multiple_files.rst

```diff
@@ -0,0 +1,7 @@
+- **Changed:**
+  `coqc` now enforces that at most a single `.v` file can be passed in
+  the command line. Support for multiple `.v` files in the form of
+  `coqc f1.v f2.v` was already quite broken due to OCaml linking state,
+  and didn't even work at all in 8.13.
```

Comment:
```suggestion
  `coqc f1.v f2.v` was already broken in 8.13.
```

Reply:
Tried to improve (removing broken), kept the reason this doesn't work.